### ISSUE TITLE
Value of "ClanSize" in ServerGameSettings.json not preserving value.

### ISSUE THREAD

**Issue by tempusthales** [StunlockStudios/vrising-dedicated-server-instructions#29]
If I change the value of "ClanSize" in `ServerGameSettings.json` from 4 to something else (5, 6 or whatever), the value that shows in the server browser is still "4".

I have tried to change the variable "GameSettingsPreset" variable in `ServerHostSettings.jso`n to blank, but it still doesnt help with the problem, even so other people have suggested to set it like this "GameSettingsPreset": "StandardPvP", you have to modify `StandardPvP.json` in the GameSettingPresets folder.

If "GameSettingsPreset": "StandardPvE_Hard", you have to modify `StandardPvE_Hard.json`, etc...

This is cumbersome and shouldnt need to work this way.  Please fix.

**Comment by IceOfWraith** [comment]
+1 for this issue as well. It'd be nice to get clarification on how this should function since it doesn't seem to update on dedicated servers.

**Comment by Ponjimon** [comment]
Hm, for me it worked fine when the preset value was blank. How did you set it, to a blank string? Are the game settings in the same folder as the host settings? It helps a lot to check the server logs for errors or to see which config file it has loaded on startup

**Comment by Timond** [comment]
I can even see in the server logs that it loaded ClanSize: 6, but in-game it still limits the ClanSize to 4, even though the StandardPVP template doesn't show any override for ClanSize?

```{
  "GameModeType": "PvP",
  "CastleDamageMode": "TimeRestricted",
  "CastleHeartDamageMode": "CanBeDestroyedOnlyWhenDecaying",
  "DeathContainerPermission": "Anyone",
  "RelicSpawnType": "Unique",
  "BloodBoundEquipment": true,
  "PlayerDamageMode": "Always",
  "PvPProtectionMode": "Short",

  "CastleStatModifiers_Global": {
    "CastleLimit": 1
  }
}```

**Comment by tempusthales** [comment]
> Hm, for me it worked fine when the preset value was blank. How did you set it, to a blank string? Are the game settings in the same folder as the host settings? It helps a lot to check the server logs for errors or to see which config file it has loaded on startup

yes I looked at the server logs and no errors were detected.  The issue happens when you set your dedicated server as something other than StandardPVP.  So for example if you set it as StandardPVP_Hard, then it ingnores ClanSize and defaults to 4.

**Comment by tempusthales** [comment]
> I can even see in the server logs that it loaded ClanSize: 6, but in-game it still limits the ClanSize to 4, even though the StandardPVP template doesn't show any override for ClanSize?

That is correct.  I have tried modifying the StandardPVP_Hard.json to include ClanSize: 5 but it still defaults to 4:

```json
{
  "GameModeType": "PvP",
  "CastleDamageMode": "TimeRestricted",
  "CastleHeartDamageMode": "CanBeDestroyedByPlayers",
  "DeathContainerPermission": "Anyone",
  "RelicSpawnType": "Unique",
  "BloodBoundEquipment": true,
  "PlayerDamageMode": "Always",
  "PvPProtectionMode": "Short",
  "ClanSize": 5,

  "CastleStatModifiers_Global": {
    "CastleLimit": 2
  },

  "DropTableModifier_General": 1.25,
  "DropTableModifier_Missions": 1.25,
  "MaterialYieldModifier_Global": 1.0,
  "BloodEssenceYieldModifier": 1.0,

  "UnitStatModifiers_Global": {
    "MaxHealthModifier": 1.25,
    "PowerModifier": 1.25
  },
  "UnitStatModifiers_VBlood": {
    "MaxHealthModifier": 1.25,
    "PowerModifier": 2
  }
}
```
Its getting frustrating because I am trying to set a special server configuration and the clan size is not setable.

**Comment by fishd72** [comment]
In the folder _installDir\\_**VRisingServer_Data\StreamingAssets\Settings**_ I have two files; **ServerHostSettings.json** and **ServerGameSettings.json**

In **ServerHostSettings.json** I have the line: _"GameSettingsPreset" : ""_
In **ServerGameSettings.json** I have the line: _  "ClanSize": 10,_

This works for me, and shows my max clan size is 10.

**Comment by tempusthales** [comment]
Have you tried doing this while your server is set to StandardPvP_Hard?

I think I tried your method once but it didn't work, I'll try it again though...
